### PR TITLE
@l2succes => Tooltip QA

### PR DESCRIPTION
**Pull request by eessex**
Adds Artsy.ContextProvider to send CurrentUser to articles
Checks if isMobile before showing tooltips

### CODE REVIEW

File: src/desktop/apps/article/components/__tests__/InfiniteScrollArticle.test.js [hunk 1]
```diff
@@ -35,42 +36,41 @@ describe('<InfiniteScrollArticle />', () => {
   let InfiniteScrollArticle = rewire.default
   const { Article } = require('reaction/Components/Publishing')
 
+  const getWrapper = props => {
+    return mount(
+      <ContextProvider currentUser={null}>
```
Comment (l2succes) [inline src/desktop/apps/article/components/__tests__/InfiniteScrollArticle.test.js]: 👍

File: src/desktop/apps/article/components/App.js [hunk 2]
```diff
@@ -51,7 +53,9 @@ export default hot(module)(
       return (
         <Fragment>
           <EditPortal article={article} />
-          {this.getArticleLayout()}
+          <ContextProvider currentUser={sd.CURRENT_USER}>
```
Comment (l2succes) [inline src/desktop/apps/article/components/App.js]: Is this the most top-level entrypoint for `Article`?

Comment (eessex) [reply src/desktop/apps/article/components/App.js]: Yes, this is rendered from here: https://github.com/artsy/force/blob/master/src/desktop/apps/article/routes.js#L140

**Comment by eessex** [comment]
@l2succes ok to merge?

**Pull request merged**